### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Azonkeu**
Hi there👋!

- [ ] This pull request is to create the Bookstore App, set up the linters, and create the components.

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        color: 'blue',
+        fontWeight: 'bold',
+        fontSize: '1.1em',
+        marginLeft: '4%',
+      }}
+      >
+        Economy
+      </li>
+    </ul>
+  </div>
```
Comment (bimbolabuari) [inline src/components/Categories.js]: As required, `Categories.js` should display "Under construction" text only

image

Kindly fix this 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Azonkeu , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Correct Gitflow
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two `<Route>`s and `<Link>`s for the app's navigation:
- Books - the default view
  - Display the list of books with the Remove button (no functionality yet)
  - Have a form for adding a book (no functionality yet)

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/components/Categories.js (hunk 1 above)
Comment (Azonkeu) [reply src/components/Categories.js]: Please what does it mean?

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @Azonkeu, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**